#request
int32 caller
string command
---
#response
int32 status
